src/ServicePulse.Host/app/modules/monitoring/views/endpoint_details.html

```diff
@@ -253,6 +253,8 @@ <h5 ng-class="{active: showInstancesBreakdown}">
                         </div>
                     </div>
                 </div>
+                
+                <no-data ng-show="endpoint.instances.length == 0" message="No messages processed in this period of time."></no-data>
```

Comment:
We have a `no-data` component?? ![](http://www.reactiongifs.com/wp-content/uploads/2011/09/mind_blown.gif)